vkcs/db_backup.go

```diff
@@ -0,0 +1,26 @@
+package vkcs
+
+import (
+	"fmt"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func databaseBackupStateRefreshFunc(client databaseClient, backupID string) resource.StateRefreshFunc {
+	return func() (interface{}, string, error) {
+		b, err := dbBackupGet(client, backupID).extract()
+		if err != nil {
+			if _, ok := err.(gophercloud.ErrDefault404); ok {
+				return b, "DELETED", nil
```

Comment:
could we use a named constant here?

Reply:
I think we could, but this style (without named constants) is used everywhere in similar methods of provider, thus we will need to replace code in many places to maintain similarity.

Reply:
ok